cmd/opm/alpha/veneer/render.go

```diff
@@ -0,0 +1,83 @@
+package veneer
+
+import (
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+	"github.com/operator-framework/operator-registry/alpha/veneer/basic"
+	"github.com/operator-framework/operator-registry/cmd/opm/internal/util"
+	containerd "github.com/operator-framework/operator-registry/pkg/image/containerdregistry"
+)
+
+func newBasicVeneerRenderCmd() *cobra.Command {
+	var (
+		veneer basic.Veneer
+		output string
+	)
+	cmd := &cobra.Command{
+		Use:   "render veneer-file",
```

Comment:
Given my earlier comment about supporting multiple veneers, should we have the `opm alpha veneer` subcommand names be the names of the veneers?

That way, when we add the `semver` veneer, we'd end up with `opm alpha veneer basic` and `opm alpha veneer semver`.

```suggestion
		Use:   "basic <veneer-file>",
```